ompi/mca/mtl/portals4/mtl_portals4_send.c

```diff
@@ -40,7 +40,7 @@ ompi_mtl_portals4_callback(ptl_event_t *ev,
                            ompi_mtl_portals4_base_request_t* ptl_base_request,
                            bool *complete)
 {
-    int retval = OMPI_SUCCESS, ret, val, add = 1;
+    int retval = OMPI_SUCCESS, ret = 0, val = 0, add = 1;
```

Comment:
My understanding is that this will set the variables and then set retval to 1 (the result of add = 1) rather than OMPI_SUCCESS (comma as an operator should evaluate but discard the first input and evaluate and use the second, if I remember correctly). I could be missing something though?

Reply:
@mdosanjh Thanks for the review.  In this case, the comma is acting as a separator not an operator so each variable keeps its initialization value.